MANIFEST.in

```diff
@@ -1,6 +1,6 @@
 include notifications_utils/email_preview_template.jinja2
 include notifications_utils/email_template.jinja2
 include notifications_utils/letter_pdf_template.jinja2
-include notifications_utils/letter_preview_template.jinja2
+include notifications_utils/letter_image_template.jinja2
```

Comment:
or even `recursive-include notification_utils *.jinja2` in case we move the templates into a subdir